src/IssueViz.Security/HotspotsList/HotspotsControl.xaml

```diff
@@ -256,7 +257,7 @@
                     CommandParameter="{Binding Hotspots/}"/>
             </DataGrid.InputBindings>
             <DataGrid.Columns>
-                <DataGridTemplateColumn CanUserSort="False" CanUserResize="False" Width="30">
+                <DataGridTemplateColumn CanUserSort="False" CanUserResize="False" Width="25">
```

Comment:
Can be slightly smaller and still show the icon - leaves more space for the variable-length columns.

Reply:
It [used to be 25](https://github.com/SonarSource/sonarlint-visualstudio/pull/1964#discussion_r534268798) and it is ever so slightly cut:
![image](https://user-images.githubusercontent.com/60586879/101140727-a5456880-3613-11eb-92f8-57bbf14dabbe.png)

Also, I see now that the icons are not aligned:
![image](https://user-images.githubusercontent.com/60586879/101140773-b3938480-3613-11eb-9f79-00fe76e03a5d.png)

They never aligned due to the way they are in VS so I added a margin on one of them to sync them. But that margin is now out of date due to the column change. So for width 25 I'd change the icon's margin to -2.5.

I did some experiments, I think width 27 and margin -3.5 is looking good as well. The icon is cut by 1-2 pixels and it's hardly noticeable:
![image](https://user-images.githubusercontent.com/60586879/101141432-97441780-3614-11eb-971f-18d152242d0b.png)

Reply:
Hhmm. Thanks for trying those variations. I'll switch it back to 30 to avoid it being cut for now. We can tweak it again later if/when we add a help pane.